plans/network/pingpong.go

```diff
@@ -60,9 +60,8 @@ func pingpong(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 
 	config.IPv4 = runenv.TestSubnet
 	config.IPv4.IP = append(config.IPv4.IP[0:2:2], ipC, ipD)
-	// Translates to /15 mask. The mask needs to match the IP range of the TestSubnet,
-	// otherwise some addresses may be excluded, causing the test to fail
-	config.IPv4.Mask = []byte{255, 254, 0, 0}
+	// the mask should match the data network IP range (e.g. for EKS it is /12)
+	config.IPv4.Mask = []byte{255, 240, 0, 0}
 	config.CallbackState = "ip-changed"
```

Comment:
blocked by https://github.com/testground/testground/pull/1462#discussion_r985597794

Reply:
(feel free to mark as resolved if we fixed it)